lucille-core/src/test/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnectorTest.java

```diff
@@ -44,6 +48,7 @@ public class DatabaseConnectorTest {
   @Rule
   public final DBTestHelper dbHelper = new DBTestHelper("org.h2.Driver", "jdbc:h2:mem:test", "",
       "", "db-test-start.sql", "db-test-end.sql");
+  private static TimeZone originalTimeZone;

```

Comment:
i think you can make this final and initialize it to TimeZone.getDefault() all on this line. Then you don't need to do it in setUpTimeZone()

Reply:
done in [26fc6fc](https://github.com/kmwtechnology/lucille/pull/184/commits/26fc6fc18d34beada13380cc47717f51ed6b7d9d)